module/config.mjs

```diff
@@ -985,10 +985,20 @@ preLocalize("itemProperties", { keys: ["label", "abbr"], sort: true });
  * @enum {object}
  */
 DND5E.validProperties = {
+  backpack: new Set([
+    "mgc"
+  ]),
+  equipment: new Set([
```

Comment:
Consumables should be able to be marked as magical also.

Reply:
Not sure what you would suggest we do here. The set for `consumable` is at the bottom (should probably be alphabetical, yes), but the properties apply only if the subtype is `ammo`.

Reply:
Aw, that's why I didn't see it when I was looking at potions in particular. Specifically I was thinking it would be good to be able to mark potions as magical, but because ammo needs additional properties that is tough with this structure. Maybe display properties for all consumable types, but filter out the `isPhysical` properties unless the consumable type is ammo?

Reply:
Alternatively we could come up with a more generalized approach for subtypes and in the meantime suffer being able to make potions made of adamantine. This would require that all item types that have `system.properties` also have `system.type` in their schema.

Reply:
Afraid I'm not seeing a good solution here since `mgc` is also `isPhysical`.